autoload/airline/extensions/tabline/buffers.vim

```diff
@@ -126,6 +133,15 @@ function! airline#extensions#tabline#buffers#get()
   return s:current_tabline
 endfunction
 
+" Compatibility wrapper for strchars, in case current vim version does not have it natively
+function! s:strchars(str)
+  if exists('*strchars')
+    return strchars(a:str)
+  else
+    return strlen(substitute(a:str, '.', 'a', 'g'))
+  endif
+endfunction
+
```

Comment:
Hm, perhaps vim-airline should require at least a vim 7.3 (which introduced the `strchars()` function)

Reply:
Yes, the reason I added this wrapper was in order to support vim 7.2 as mentioned in the contribution guide for the project... (It's surprising how many things in my vimrc don't support vim 7.2 anymore, I had to remove a ton of things to be able to test these changes on 7.2).

Reply:
okay, good.